tests/test_dataset.py

```diff
@@ -1057,6 +1060,18 @@ def test_preprocess_replace_zero(self):
             self.obj.preprocessing_info[PreprocessingStateKeys.REPLACE_ZEROES], True
         )
 
+    def test_create_id_mapping(self):
+        """Test id maps"""
+        self.assertEqual(len(self.obj._gene_to_features_map), 21)
```

Comment:
could we test for the explict maps here?

Reply:
Sure can do.